File: despatch.WEB/app/src/swagger/models/user-organizations-response.ts
/* tslint:disable */
import { BaseEntityResponse } from './base-entity-response';
import { OrganizationResponse } from './organization-response';
import { UserResponse } from './user-response';
export interface UserOrganizationsResponse extends BaseEntityResponse{
  contactDataId?: string;
  organization?: OrganizationResponse;
  organizationId?: string;
  user?: UserResponse;
}

File: despatch.Domain/Models/Default/Response/UserOrganizationsResponse.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace despatch.Domain.Models.Default.Response
{
    public class UserOrganizationsResponse : BaseEntityResponse
    {
        public string ContactDataId { get; set; }
        public string OrganizationId { get; set; }

        public virtual UserResponse User { get; set; }
        public virtual OrganizationResponse Organization { get; set; }
    }
}
